bot/resources/tags/print-return.md

```diff
@@ -0,0 +1,10 @@
+---
+embed:
+    image:
+        url: https://cdn.discordapp.com/attachments/267659945086812160/998198889154879558/print-return.gif
+---
+**Print and Return**
```

Comment:
```suggestion
---
embed:
    title: "Print and Return"
    image:
        url: https://cdn.discordapp.com/attachments/267659945086812160/998198889154879558/print-return.gif
---
```
I think a proper title looks better; it's a bit more distinct and doesn't add the big spacing https://i.imgur.com/BRR78id.png